src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -34,7 +34,7 @@ static SpanCollector GetInstance(Uri uri, uint maxProcessorBatchSize)
             return instance;
         }
 
-        public ZipkinClient(IZipkinConfig zipkinConfig, IOwinContext context, SpanCollector collector = null)
+        public ZipkinClient(IZipkinConfig zipkinConfig, HttpContextBase context, SpanCollector collector = null)
```

Comment:
Just want to confirm the adhoc if it covered these scenarios:
1. For Owin application, is zipkin properly logging for both incoming and outgoing?
2. For Web API application, is zipkin properly logging for both incoming and outgoing?